Translated/Story/13 The Power of Song/msn_012245.csv

```diff
@@ -0,0 +1,3 @@
+name_7018#0,"""Jalil"""
+msn_012245_0010#0,"""We will be routing the DOLLS hindering our route to Lenus Retem."""
```

Comment:
I can't put my finger on what it is, but something about this line doesn't sit right with me.

Reply:
"We'll be diverting the DOLLS that are hindering our route to Lenus Retem".

Reply:
How about "obstructing our route"?